examples/pixel/.eslintrc.json

```diff
@@ -5,6 +5,7 @@
   ],
   "rules": {
     "camelcase": "off",
+    "react/jsx-props-no-spreading": "off",
```

Comment:
I sometime just hate the arbitrary opinions in eslint, this is one of them.

Reply:
Yes, I need to switch this to one of the standard eslint configs, haven't gotten around to it yet, but I will in the next couple of weeks